r1cs/gadgets/std/src/fields/nonnative/tests.rs

```diff
@@ -59,7 +59,7 @@ fn ceil_log_2_biguint_test() {
     }
```

Comment:
Can we add, if possible, tests that compares mul (and maybe also reduce) done with and without mersenne, and assert that the results are the same ?

Reply:
I added the test for pseudo-mersenne fields

Reply:
We have already tests that compares with primitive value so forget it